### ISSUE TITLE
broken presm_ links

### ISSUE THREAD

**Issue by davidcarlisle** [w3c/mathml#324]
The last PR removed the section

```
<section>
<h5 id="presm_mixtextmath">Mixing text and mathematics</h5>

<p>In some cases, text embedded in mathematics could be more appropriately
represented using <code class="element">mo</code> or <code class="element">mi</code> elements.
```

but it is referenced in two places

```
    <h5 id="presm_html_insertion">Embedding HTML in MathML</h5>
 
    <p>MathML can be combined with other formats as described in
    <a href="#world-int-combine-other"></a>.
    The recommendation is to embed other formats in MathML by extending the MathML
    schema to allow additional elements to be children of the <code class="element">mtext</code> element or
    other leaf elements as appropriate to the role they serve in the expression
    (see  <a href="#presm_mixtextmath"></a>).
```

and

```
  <section>
   <h4 id="presm_mi">Identifier <code class="defn starttag">&lt;mi&gt;</code> <a class="coreyes" href="https://w3c.github.io/mathml-core/#dfn-mi"></a></h4>
 
   ...
 
   Conversely, arbitrary text
    playing the role of a <span>&#x201c;term&#x201d;</span> (such as an ellipsis in a summed series)
    can be represented using an <code class="element">mi</code> element, as shown
    in an example in <a href="#presm_mixtextmath"></a>.</p>
```

**Comment by NSoiffer** [comment]
I moved that section to the notes because it was informative/commentary.
I'll look at the references and see if they are really needed (e.g., maybe
just add the example directly).

**Comment by davidcarlisle** [comment]
fixed by #327
